netket/stats/mc_stats.py

```diff
@@ -0,0 +1,139 @@
+from numba import jit
+import numpy as _np
+from . import mean as _mean
+from . import var as _var
+from . import total_size as _total_size
+
+
+class Stats:
+    """A dict-compatible class containing the result of the statistics function."""
+
+    _NaN = float("NaN")
+
+    def __init__(
+        self, mean=_NaN, error_of_mean=_NaN, variance=_NaN, tau_corr=_NaN, R=_NaN,
+    ):
+        self.mean = mean
+        self.error_of_mean = error_of_mean
+        self.variance = variance
+        self.tau_corr = tau_corr
+        self.R = R
+
+    def to_json(self):
+        jsd = {}
+        jsd["Mean"] = self.mean.real
+        jsd["Variance"] = self.variance
+        jsd["Sigma"] = self.error_of_mean
+        jsd["R"] = self.R
+        jsd["TauCorr"] = self.tau_corr
+        return jsd
```

Comment:
unfortunately complex values are to json serializable, thus the need for a converter here

Reply:
Well it makes more sense to return this dictionary, anyway. It's just not JSON at this point, it will only be if someone calls, e.g.,
```
import json
json.dumps(stats.to_json())
```
on it. So really, `asdict` is a more fitting name.